src/patternfly/components/Menu/menu.scss

```diff
@@ -627,6 +628,10 @@
   flex-grow: 1;
 }
 
+.pf-c-menu__item-text.pf-m-danger {
```

Comment:
If you move this selector to `__list-item`, you can set this color by overriding the text color var in `__list-item&.pf-m-danger`. You'll just want to make sure it plays nicely with the disabled variation (text should be light grey if the item is danger and disabled)

`--pf-c-menu__list-item--Color: --pf-c-menu__list-item--m-danger__item--Color;`

Reply:
Applied update related to this edit, thanks